node/cli/src/chain_spec.rs

```diff
@@ -109,9 +109,8 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 			bonding_duration: 60 * MINUTES,
 			offline_slash_grace: 4,
 			minimum_validator_count: 4,
-			stakers: initial_authorities.iter().map(|x| (x.0.into(), x.1.into(), STASH)).collect(),
+			stakers: initial_authorities.iter().map(|x| (x.0.into(), x.1.into(), STASH, StakerStatus::Validator)).collect(),
```

Comment:
@gavofyork is this ok or should be more generic?

Reply:
looks reasonable. how could it be more generic?

Reply:
Just that using a data type deep from a module in here seemed a bit strange. Though it is also used directly inside `build()`; it makes sense.